src/gui/networksettings.cpp

```diff
@@ -155,12 +156,25 @@ void NetworkSettings::saveProxySettings()
     } else if (_ui->systemProxyRadioButton->isChecked()) {
         cfgFile.setProxyType(QNetworkProxy::DefaultProxy);
     } else if (_ui->manualProxyRadioButton->isChecked()) {
-        int type = _ui->typeComboBox->itemData(_ui->typeComboBox->currentIndex()).toInt();
-        bool needsAuth = _ui->authRequiredcheckBox->isChecked();
-        QString user = _ui->userLineEdit->text();
-        QString pass = _ui->passwordLineEdit->text();
-        cfgFile.setProxyType(type, _ui->hostLineEdit->text(),
-            _ui->portSpinBox->value(), needsAuth, user, pass);
+
+        QString hostLineEditValue = _ui->hostLineEdit->text();
+
+        if(hostLineEditValue.isEmpty())
+        {
+            _ui->hostLineEdit->setStyleSheet("border: 1px solid red");
+            return;
+        }
+        else
+        {
+            _ui->hostLineEdit->setStyleSheet("border: 1px solid black"); // or blue
```

Comment:
did you mean  "border: native" ?
or just `_ui->hostLineEdit->setStyleSheet(QString())`

Reply:
Yes, thank you. "border: native" is very good.